src/manual.md

```diff
@@ -2,89 +2,91 @@
-notice. [Bug reports](https://github.com/denoland/deno/issues) do help!
-
-## Introduction
+**A word of caution: Deno is very much under development.**
 
-A secure JavaScript/TypeScript runtime built with V8, Rust, and Tokio
-
-### Philosophy
-
-Deno aims to be a productive and secure scripting environment for the modern
-programmer.
-
-Deno will always be distributed as a single executable. Given a URL to a Deno
-program, it is runnable with nothing more than
-[the 10 megabyte zipped executable](https://github.com/denoland/deno/releases).
-Deno explicitly takes on the role of both runtime and package manager. It uses a
-standard browser-compatible protocol for loading modules: URLs.
-
-Deno provides security guarantees about how programs can access your system with
-the default being the most restrictive secure sandbox.
+We encourage brave early adopters, but expect bugs large and small. The API is subject to change without
+notice. [Bug reports](https://github.com/denoland/deno/issues) do help!
 
-Deno provides <a href="https://github.com/denoland/deno/tree/master/std">a set
-of reviewed (audited) standard modules</a> that are guaranteed to work with
-Deno.
+We are [actively working towards 1.0](https://github.com/denoland/deno/issues/2473), but there is no date guarantee.
 
-### Goals
+## Introduction
 
-- Support TypeScript out of the box.
+Deno is a JavaScript/TypeScript runtime with secure defaults and a great developer experience.
 
-- Uses "ES Modules" and does not support `require()`. Like the browser, allows
-  imports from URLs:
+It's built on V8, Rust, and Tokio.
 
-  ```ts
-  import * as log from "https://deno.land/std/log/mod.ts";
-  ```
+### Feature Highlights
 
-- Remote code is fetched and cached on first execution, and never updated until
-  the code is run with the `--reload` flag. (So, this will still work on an
-  airplane. See `~/.deno/src` for details on the cache.)
+* Secure by default. No file, network, or environment access (unless explicitly enabled).
+* Supports TypeScript out of the box.
+* Ships a single executable (`deno`).
+* Has built in utilities like a dependency inspector (`deno info`) and a code formatter (`deno fmt`).
+* Has [a set of reviewed (audited) standard modules](https://github.com/denoland/deno/tree/master/std) that are guaranteed to work with Deno.
+* Scripts can be bundled into a single javascript file.
 
-- File system and network access can be controlled in order to run sandboxed
-  code. Access between V8 (unprivileged) and Rust (privileged) is only done via
-  serialized messages. This makes it easy to audit. For example, to enable write
-  access use the flag `--allow-write` or for network access `--allow-net`.
+##### On the Roadmap
 
-- Only ship a single executable.
+* Native modules in Rust.
+* Scripts can be bundled into a binary with the deno runtime.
+  * This makes distribution simple and script startup very fast.
```

Comment:
I created a new section called "Feature Highlights" which helps clean up some of the items in the goal section that weren't really goals per se.

I also think this is an important spot to pitch newcomers on our key features.

I added "Native modules in Rust" because I believe that's a goal and it's 🔥.

I added "Scripts can be bundled into a binary with the deno runtime" because it's also a feature we are pursuing and is a great differentiator.